# This file is part of HiGFXback

# requires
REQUIRES="autotools-wrappers-lt-build libxext-build libxrender-build randrproto-build"

PKG_CONFIG_PATH=/x11/share/pkgconfig pkg-config --exists --print-errors $REQUIRES || exit 1

# configure (ac-2.63; am-1.10; lt-2.2.6)
libtoolize -c -f; aclocal -I m4; autoconf; autoheader; automake -a -c
PKG_CONFIG_PATH=/x11/lib/pkgconfig LDFLAGS=-Wl,-rpath,/x11/lib ./configure --disable-static --prefix=/x11

# build
make

# install
install -d $DESTDIR/x11/include/X11/extensions
install -m 644 include/X11/extensions/Xrandr.h $DESTDIR/x11/include/X11/extensions
install -d $DESTDIR/x11/lib
install src/.libs/libXrandr.so.2.* $DESTDIR/x11/lib/libXrandr.so.2
ln -sf libXrandr.so.2 $DESTDIR/x11/lib/libXrandr.so
install -d $DESTDIR/x11/lib/pkgconfig
install -m 644 xrandr.pc $DESTDIR/x11/lib/pkgconfig

# build.pc
install -d $DESTDIR/x11/share/pkgconfig
cat > $DESTDIR/x11/share/pkgconfig/libxrandr-build.pc << EOF
Name: libXrandr
Version: 1.3.0
Description: X resize and rotate library
Requires: $REQUIRES

devel=\\
/x11/include/X11/extensions/Xrandr.h \\
/x11/lib/libXrandr.so \\
/x11/lib/pkgconfig/xrandr.pc

exec=\\
/x11/lib/libXrandr.so.2
EOF

exit
--- libXrandr-1.3.0.orig/configure.ac
+++ libXrandr-1.3.0/configure.ac
@@ -32,6 +32,7 @@
 dnl
 AC_INIT(libXrandr, 1.3.0, [https://bugs.freedesktop.org/enter_bug.cgi?product=xorg], libXrandr)
 AC_CONFIG_AUX_DIR(.)
+AC_CONFIG_MACRO_DIR([m4])
 AM_INIT_AUTOMAKE([dist-bzip2])
 AM_MAINTAINER_MODE
 
